### PR TITLE
From + until

### PR DESCRIPTION
**Pull request by sndrs**

### CODE REVIEW

File: src/index.ts [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
+				.splice(i + 1)
+				.reduce(
+					(untils, [untilName, untilWidth], i) => ({
+						[untilName]: fromUntilQuery(fromWidth, untilWidth),
+						...untils,
+					}),
+					{},
+				),
+			// this piece of freaky genius by @SiAdcock
+			toString: () => fromQuery(fromWidth),
```
Comment (SiAdcock) [inline src/index.ts:40]:
```
 -------------------
/     Here lies     \
| Simon John Adcock |
|  A freaky genius  |
|                   |
|  1982 - 208[x]*   |
|                   |
|                   |
```

<sub>*illegible</sub>

**Review by SiAdcock — APPROVED** [review]
Great start! 🎆

**Comment by SiAdcock** [comment]
Haha I was just investigating what was happening with CI! Thanks for clarifying 😂

File: __tests__/index.ts [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+describe('until', () => {
+	test('is an object that provides breakpoint media queries', () => {
+		expect(until.small).toBe('@media all and (max-width: 29.9375em)')
+	})
+})
+
+describe('from', () => {
+	test('is a weird old beast', () => {
+		expect(from.small.toString()).toBe('@media all and (min-width: 30em)')
```
Comment (sndrs) [inline __tests__/index.ts:11]: this bit worries me, in that unable so far to prove this actually works. it _should_ do though, right?

Comment (SiAdcock) [reply __tests__/index.ts:11]: Is this any more explicit...
```js
expect(`${from.small}`).toBe('@media all and (min-width: 30em)')
```
It's closer to the actual syntax

Comment (sndrs) [reply __tests__/index.ts:11]: i'll merge it so it's a bit easier to see everything in one go

Comment (sndrs) [reply __tests__/index.ts:11]: you end up with this:

```
    Expected: "@media all and (min-width: 30em)"
    Received: {"toString": [Function toString], "until": {"large": etc...}
```

Comment (sndrs) [reply __tests__/index.ts:11]: so question is: is this a quirk of the testing env or correct?

**Pull request merged**